components/atom/image/src/index.scss

```diff
@@ -0,0 +1,66 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bgc-atom-image: $c-gray-light !default;
+
+.sui-AtomImage {
+  height: 100%;
+  margin: 0;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+
+  &-figure {
+    background-color: $bgc-atom-image;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    margin: 0;
+  }
+
+  &--placeholder.is-loading &-figure {
+    filter: blur(5px);
```

Comment:
Am I not sure if it's really browser compatible; you could use a mixin to fallback a wide browser compatibiity solution ;-)

Reply:
looks like pretty compatible, isn't it?
https://caniuse.com/#search=filter%3A%20blur

Reply:
This `blur` effect is not part of the specification (https://paper.dropbox.com/doc/SUI-Image-Cv0S94htbykeN8cRkilPV). 

It's just a nice add that will work in most of the supported browsers according to this list: https://github.schibsted.io/scmspain/frontend-all-wiki/blob/on-boarding-guide/wiki/Components/new-component.md#browser-support

If this is not supported in some browser the final effect will still work according to the specs.